backend/graphql/index.ts

```diff
@@ -65,7 +65,7 @@ const graphQLMiddlewares = {
     simpleEntities: authorizedByAllRoles(),
     userById: authorizedByAdmin(),
     users: authorizedByAdmin(),
-    tests: authorizedByAllRoles(),
+    // tests: authorizedByAllRoles(),
```

Comment:
Hey so I'm having trouble with this line, it causes me an unauthorized error and I'm not sure why?

Reply:
i think it's bc you're using an unverified account (my old one right?). i pulled your branch & tested it locally and it isnt throwing any authorization errors for me!

try creating a new account and verifying it and then you shouldn't have any issues (: